src/components/TenantSelector/TenantSelector.tsx

```diff
@@ -250,67 +251,131 @@ const LoginWrapper = styled.div`
   -webkit-box-pack: start;
   justify-content: flex-start;
   max-width: 400px;
-  box-shadow: rgba(0, 0, 0, 0.05) 1px 6px 20px 8px,
-    rgba(0, 0, 0, 0.11) 0px 6px 6px;
-  background-color: rgb(255, 255, 255);
+  box-shadow: ${({ theme }) => theme.gearbox.lightShadow};
+  background-color: ${({ theme }) => theme.gearbox.containerColor};
   margin: auto;
   padding: 45px;
   overflow: auto;
 `;
 
+LoginWrapper.defaultProps = {
+  theme: {
+    gearbox: defaultTheme,
+  },
+};
+
 const Title = styled.h1`
   font-weight: bold;
+  color: ${({ theme }) => theme.gearbox.textColor};
+`;
+
+Title.defaultProps = {
+  theme: {
+    gearbox: defaultTheme,
+  },
+};
+
+const SubTitle = styled.h3`
+  color: ${({ theme }) => theme.gearbox.textColor};
 `;
 
-const SubTitle = styled.h3``;
+SubTitle.defaultProps = {
+  theme: {
+    gearbox: defaultTheme,
+  },
+};
 
 const Input = styled((props: InputProps) => <AntInput {...props} />)`
   && {
     border: none;
-    background-color: #f5f5f5;
+    background-color: ${({ theme }) => theme.gearbox.lightGrey};
     font-weight: bold;
     border-radius: 0;
     height: 48px;
-    border-bottom: 2px solid #f5f5f5;
+    border-bottom: 2px solid ${({ theme }) => theme.gearbox.lightGrey};
     margin-bottom: 8px;
-
+    color: ${({ theme }) => theme.gearbox.textColor};
     &:focus,
     &:active {
       box-shadow: none;
-      border-bottom: 2px solid #179aff;
+      border-bottom: 2px solid ${({ theme }) => theme.gearbox.primaryColor};
+    }
+    &:hover {
+      border-bottom: 2px solid ${({ theme }) => theme.gearbox.primaryColor};
+    }
+    &::placeholder {
+      color: ${props => props.theme.gearbox.textColorDisabled} !important;
```

Comment:
Could we have destructuring assignment here (and other place also) as we use before:
```typescript
({theme}) => theme.gearbox.buttonBorderColor
```